communities.json

```diff
@@ -9,7 +9,7 @@
     "https://karte.freifunk-ansbach.de/data/nodes.json"
   ],
   "Bielefeld": [
-    "https://map.freifunk-bielefeld.de/meshviewer/data/meshviewer.json"
+    "https://map03.4830.org/data/nodes.json"
```

Comment:
This nodes.json reveals 566 nodes, while Bielefeld has 87?

Reply:
the json is also already included as gütersloh

https://github.com/freifunk-gluon/census-exporter/blob/895e2fd4c7b9d1afee092464f31f92cf3331cf93/communities.json#L81-L83

Reply:
Oh well, then it is probably better to separate the communities correctly as given here:
https://map03.4830.org/

@herbetom weird that I missed that..

Reply:
I don't quite see the point in seperating them. From my understanding it's not like their using different Firmwares.

Reply:
@herbetom yes thats true, but separating them would improve just the naming.
Having all those communities under the name "Gütersloh" seems weird too.

What would you suggest instead?
Renaming to Gütersloh/Bielefeld or 4830.org ?

Reply:
Splitting would mean having to deal with a many-fold number of timeouts in the case of problems.
In addition i don't think it's relevant for the census-exporter where exactly Nodes are located. What might matter is how you can find a map/website for the community that includes them.

So yeah, maybe 4830.org or leave it at Gütersloh. I don't think Gütersloh/Bielefeld would be the way to go.